iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -239,6 +252,16 @@ public void abortJob(JobContext originalContext, int status) throws IOException
     cleanup(jobContext, jobLocations);
   }
 
+  private Set<FileStatus> listForCommits(JobConf jobConf, String jobLocation) throws IOException {
```

Comment:
nit: Maybe a javadoc that do not use it for anything else than abort.

Reply:
Right, good idea